xdg-desktop-portal-wlr.5.scd

```diff
@@ -0,0 +1,48 @@
+xdg-desktop-portal-wlr(5)
+
+# NAME
+
+xdg-desktop-portal-wlr - an xdg-desktop-portal backend for wlroots
+
+# DESCRIPTION
+
+xdg-desktop-portal-wlr (or xdpw for short) allows applications to request
+screenshots and screencasts via xdg-desktop-portal in wlroots-based Wayland
+compositors.
+
+xdpw will try to load the configuration file from these locations:
+
+- $XDG_CONFIG_HOME/xdg-desktop-portal-wlr/$XDG_CURRENT_DESKTOP
+- $XDG_CONFIG_HOME/xdg-desktop-portal-wlr/config
+- /etc/xdg/xdg-desktop-portal-wlr/$XDG_CURRENT_DESKTOP
+- /etc/xdg/xdg-desktop-portal-wlr/config
```

Comment:
It also looks in `/etc/xdg-desktop-portal-wlr/...` too...

Reply:
Yeah, it sounds like it's a little too many paths. Maybe we should drop either of these, they serve the same purpose.

Reply:
see https://github.com/emersion/xdg-desktop-portal-wlr/pull/100#issuecomment-799388639

I added those, since i didn't know, how common they are on other unices.